yb-voyager/src/dbzm/dbzm.go

```diff
@@ -83,8 +84,16 @@ func (d *Debezium) Start() error {
 	if err != nil {
 		return err
 	}
+
+	YB_OR_PG_CONNECTOR_PATH := filepath.Join(DEBEZIUM_DIST_DIR, "pg-connector")
```

Comment:
This should work, no?

```
if targetDBExporterRole and d.Config.UseLogicalReplicationYBConnector {
		YB_OR_PG_CONNECTOR_PATH = filepath.Join(DEBEZIUM_DIST_DIR, "yb-connector")
else 
                 YB_OR_PG_CONNECTOR_PATH := filepath.Join(DEBEZIUM_DIST_DIR, "pg-connector")
```

Reply:
I think this might make it easier to read:
```
        var YB_OR_PG_CONNECTOR_PATH string
	if isTargetDBExporter(d.ExporterRole) {
		if d.Config.UseLogicalReplicationYBConnector {
			// In case of logical replication connector we need the path /opt/yb-voyager/debezium-server/yb-connector
			YB_OR_PG_CONNECTOR_PATH = filepath.Join(DEBEZIUM_DIST_DIR, "yb-connector")
		} else {
			// In case of gRPC connector the DEBEZIUM_DIST_DIR is set to debezium-server-1.9.5, hence the connector path is empty to avoid any errors in run.sh
			YB_OR_PG_CONNECTOR_PATH = ""
		}
	} else {
		// In case of source db exporter we need the path /opt/yb-voyager/debezium-server/pg-connector
		YB_OR_PG_CONNECTOR_PATH = filepath.Join(DEBEZIUM_DIST_DIR, "pg-connector")
	}
```